map/src/main/java/openlr/map/utils/GeometryUtils.java

```diff
@@ -702,6 +716,21 @@ public static GeoCoordinates determineCoordinateInDistance(
         return new GeoCoordinatesImpl(lon2, lat2);
     }
 
+    /**
+     * Degree of interval between two angles
+     * @param firstBearing angle of line to the north

```

Comment:
"angle of line to the north" makes it sound like there is a line going to the north.  Maybe "angle of a line relative to due north".

Reply:
458ee63d9b353f59b5145a2bb129eef2fe2ab6ad